lib/jnpr/junos/command/chassisethernetswitchstatistics.yml

```diff
@@ -1,6 +1,7 @@
 ---
 EthernetSwitchStatistics:
     command: show chassis ethernet-switch statistics
+    key: fpc
     view: EthernetSwitchStatisticsView
 
 EthernetSwitchStatistics:
```

Comment:
I am rejecting this pull request because the issue is different. View name and the view is not same. Please send a new pull request just for chassisethernetswitcherrors.yml

Reply:
ack - let me close this pull request.